stories/panel.stories.ts

```diff
@@ -129,4 +145,45 @@ storiesOf('Basics/Panel', module)
         }),
       ]);
     })
+  )
+  .add(
+    'collapsed unless canRefine',
+    withHits(
+      ({ search, container }) => {
+        const breadcrumbInPanel = panel({
+          collapsed: ({ widgetRenderState }) => {
+            // TODO: We need better typing here. Not sure how, yet.
```

Comment:
I don't think it's possible to get type values from the third call inside the first :(

I guess it's possible if panel was a single function call, but that's very different from the current implementation